Makefile

```diff
@@ -0,0 +1,12 @@
+clipper: target/build/clipper
```

Comment:
I think this is the wrong path. Should be `target/debug/clipper`. But getting the path correct introduces a new problem: not rebuilding when it's necessary. 

It's probably reasonable to either use dependency info files: https://doc.rust-lang.org/stable/cargo/guide/build-cache.html#dep-info-files, live example: https://github.com/lf-/mu/blob/main/Makefile#L29-36

or to set the target as `.phony`.